README.md

```diff
@@ -1,20 +1,19 @@
 <p align="center">
-  <img width="300" alt="giskardlogo" src="giskard-frontend/src/assets/logo_full.svg">
+  <img width="500" alt="giskardlogo" src="readme/Logo_full_darkgreen.png">
 </p>
-<h3 align="center">Deliver ML products, better & faster</h3>
+<h1 align="center" weight='300' >Open-Source CI/CD platform for ML teams. Deliver ML products, better & faster. </h1>
```

Comment:
I would keep the title just "Open-Source CI/CD platform for ML teams" and then have "Deliver ML products, better & faster" below the title, in bold.

Reply:
@mackerac As you can see below, having two lines below titles is odd.
![Screenshot 2022-06-22 at 10 40 37](https://user-images.githubusercontent.com/4739848/174984641-bd0ff536-d17e-4c20-8f3e-787744eabd01.png)

To fix it, can you change ""Deliver ML products, better & faster" to h3?

Reply:
Done